### PR TITLE
chore: fix link in showcase

### PR DESCRIPTION
**Pull request by luism3861**
this PR fix link in showcase for React Native open source apps, something weird happen in dev works but in production still infinite red link home page!.

cc: @Simek

### CODE REVIEW

**Review by Simek — CHANGES_REQUESTED** [review]
Hi @luism3861, unfortunately looking at the test deploy, the slash at the end of URL was not enough to fix the rehydration issue. 

Can you try to add `key` to the problematic link or even to the both links in this paragraph?

**Comment by luism3861** [comment]
> Hi @luism3861, unfortunately looking at the test deploy, the slash at the end of URL was not enough to fix the rehydration issue.
> 
> Can you try to add `key` to the problematic link or even to the both links in this paragraph?

hi @Simek i added readme key for the github link

File: website/src/pages/showcase.js [hunk 1]
```diff
@@ -99,11 +99,10 @@ const Showcase = () => {
         </a>
         <p>
           A curated list of{' '}
-          <a href="https://github.com/ReactNativeNews/React-Native-Apps">
+          <a href="https://github.com/ReactNativeNews/React-Native-Apps/#readme">
```
Comment (slorber) [inline website/src/pages/showcase.js:102]:
```suggestion
          <a key="test" href="https://github.com/ReactNativeNews/React-Native-Apps">
```

Try this instead

Comment (Simek) [reply website/src/pages/showcase.js:102]: As Sebastien suggested, I have different `key` in mind, sorry for not being precise enough!

```suggestion
          <a key="app-list" href="https://github.com/ReactNativeNews/React-Native-Apps/#readme">
```
But please use something meaningful, so we do not need to push another commit and redeploy it, if the addition of `key` works

Comment (luism3861) [reply website/src/pages/showcase.js]: > Try this instead

@slorber  thank you for the suggestions :),

Comment (luism3861) [reply website/src/pages/showcase.js]: > > Try this instead
> 
> @slorber thank you for the suggestion :),

Comment (luism3861) [reply website/src/pages/showcase.js]: > As Sebastien suggested, I have different `key` in mind, sorry for not being precise enough!
> 
> But please use something meaningful, so we do not need to push another commit and redeploy it, if the addition of `key` fixes the issue.
@Simek  no problem if i use `demo-apps` for the `key`? i pushed my last changes

Comment (Simek) [reply website/src/pages/showcase.js]: `demo-apps` is fine, thank you for the updates! 🙂

Comment (luism3861) [reply website/src/pages/showcase.js]: thanks @Simek  !.

File: website/src/pages/showcase.js [hunk 2]
```diff
@@ -99,11 +99,12 @@ const Showcase = () => {
         </a>
         <p>
           A curated list of{' '}
-          <a href="https://github.com/ReactNativeNews/React-Native-Apps">
+          <a
```
Comment (slorber) [inline website/src/pages/showcase.js:102]: try to wrap this link in a <span> 😅

File: website/src/pages/showcase.js [hunk 3]
```diff
@@ -99,11 +99,12 @@ const Showcase = () => {
...
+            key="demo-apps"
```
Comment (slorber) [inline website/src/pages/showcase.js:103]: doesn't seem to work

try wrapping this link in a `<span>` instead 🤪 generally changing slightly the markup can solve these weird hydration issues

Comment (luism3861) [reply website/src/pages/showcase.js]: thank you so much  :) !.

**Review by Simek — APPROVED** [review]
LGTM! @luism3861 thank you for the all attempts and updates, @slorber thank you for the background on the issue and hints for the fixes! Great teamplay 👍

**Pull request merged**